News/Main.py

```diff
@@ -55,14 +67,79 @@ def App():
                 YesorNo = int(raw_input("Press 1 to save else press 0 to continue >>> "))
                 if YesorNo == 1:
                     extr.FileSave()
-                    
 
+def Topic():
+    newsTopics = NewsTopics()
+
+    while True:
+        for i in xrange(len(newsTopics)):
+            print ("[" + str(i) + "]" + "\t" + newsTopics[i])
+        print ("Please enter the index of the news topic")

```

Comment:
move these print statement into some single function called disclaimer or anything that is intuitive.